webhook-service/lib/common.go

```diff
@@ -1,7 +1,56 @@
 package lib
 
-import "os"
+import (
+	"os"
+	"strings"
+)
+
+const (
+	WebhookConfigMap        = "keptn-webhook-config"
+	KubernetesSvcHostEnvVar = "KUBERNETES_SERVICE_HOST"
+	KubernetesAPIPortEnvVar = "KUBERNETES_SERVICE_PORT"
+)
 
 func GetNamespaceFromEnvVar() string {
 	return os.Getenv("POD_NAMESPACE")
 }
+
+func GetEnv() map[string]string {
```

Comment:
let's add a unit test for this guy

Reply:
the method is there for a longer time without tests, just moved it to common.go and to be honest, do not really know how to test this kind of behaviour... It may need some test, but will definitely need a refactoring as well as the rest of webhook-service, but that is not the point of this issue and PR. If you agree I would rather skip adding tests to functionality which was not modified, as that is another story...